frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -336,22 +337,40 @@ Feature: Pipeline Runs
               And user navigates to pipelineRun logs tab
              Then user is able to see log snippet for failure of "build-image" task
 
-        @regression
+        @regression @odc-4793
         Scenario: Pipeline Run details page with Parameters tab and no parameters: P-07-TC34
             Given pipeline run is displayed for "pipeline-run-no-parameters" without resource
               And user is at pipelines page
-            When user clicks Last Run value of "pipeline-run-no-parameters"
-            Then user will be redirected to Pipeline Run Details page
+             When user clicks Last Run value of "pipeline-run-no-parameters"
+             Then user will be redirected to Pipeline Run Details page
               And user is able to see Details, YAML, TaskRuns, Parameters, Logs and Events tabs
               And user navigates to pipelineRun parameters tab
               And user is able to see No parameters are associated with this PipelineRun
 
-        @regression
+        @regression @odc-4793
         Scenario: Pipeline Run with parameters: P-07-TC35
             Given pipeline run is displayed for "pipeline-run-parameters" with parameters
               And user is at pipelines page
-            When user clicks Last Run value of "pipeline-run-parameters"
-            Then user will be redirected to Pipeline Run Details page
+             When user clicks Last Run value of "pipeline-run-parameters"
+             Then user will be redirected to Pipeline Run Details page
               And user navigates to pipelineRun parameters tab
               And user is able to see parameters of pipelineRun
               And user is able to see name "testName" and value "testValue" parameters value of pipelineRun
+
+        @regression @odc-4793 @to-do
+        Scenario: Cancelled status for the pipeline: P-07-TC34
+            Given pipeline "pipeline-cancel" is present on Pipelines page
+              And user is at the Pipeline Run page of pipeline "pipeline-cancel"
+             When user selects "Start" option from kebab menu for pipeline "pipeline-cancel"
```

Comment:
As per the Given step user is at the PipelineRun page. Then it is not possible to select the `Start` option from the Kebab menu of Pipeline.

Reply:
Updated